Cargo.lock

```diff
@@ -4174,7 +4189,8 @@ dependencies = [
  "schemars_derive",
  "serde",
  "serde_json",
- "uuid",
+ "uuid 0.8.2",
+ "uuid 1.8.0",
```

Comment:
Any idea why `schemars` is depending on two different `uuid` versions?

Reply:
[lol here we are](https://github.com/GREsau/schemars/blob/master/schemars/Cargo.toml#L26C1-L27)